frontend/rust-lib/flowy-server/src/af_cloud/impls/user/cloud_service_impl.rs

```diff
@@ -544,7 +542,11 @@ where
     FutureResult::new(async move {
       let client = try_get_client?;
       client
-        .cancel_subscription(&workspace_id, &SubscriptionPlan::Pro)
```

Comment:
I had to make a change here as this API changed in latest appflowy-cloud client-api. cc @LucasXu0

Reply:
Your changes are fine, it will be resolved by #5741